src/test/java/com/exasol/ExaOperationInterface.java

```diff
@@ -33,34 +33,56 @@ public ExaOperationInterface(final String exasolIpAddress, final String adminUse
+    public void setBucketPasswords(final String readPassword, final String writePassword) {
+        try {
+            this.client.execute("bfsdefault.default.editBucketFSBucket",
+                    new Object[] { Map.of("read_password", readPassword, "write_password", writePassword) });
+        } catch (final XmlRpcException exception) {
+            throw new IllegalStateException(
+                    "Failed to set BucketFS password in ExaOperation. Cause: " + exception.getMessage(), exception);
+        }
     }
 
     public void createAndUploadJdbcDriver(final String name, final String jdbcMainClass, final String prefix,
-            final boolean disableSecurityManager, final File driverJar) throws XmlRpcException, IOException {
-        final String driverId = addJdbcDriver(name, jdbcMainClass, prefix, disableSecurityManager);
-        uploadJdbcDriver(driverId, driverJar);
+            final boolean disableSecurityManager, final File driverJar) throws IOException {
+        try {
+            final String driverId = addJdbcDriver(name, jdbcMainClass, prefix, disableSecurityManager);
+            uploadJdbcDriver(driverId, driverJar);
+        } catch (final XmlRpcException exception) {
+            throw new IllegalStateException("Failed create or upload JDBC driver. Cause: " + exception.getMessage(),
+                    exception);
+        }
     }
 
     private String addJdbcDriver(final String name, final String jdbcMainClass, final String prefix,
             final boolean disableSecurityManager) throws XmlRpcException {
-        return (String) this.client.execute("addJDBCDriver", new Object[] { Map.of("jdbc_main", jdbcMainClass,
-                "jdbc_name", name, "jdbc_prefix", prefix, "disable_security_manager", disableSecurityManager) });
+        try {
+            return (String) this.client.execute("addJDBCDriver", new Object[] { Map.of("jdbc_main", jdbcMainClass,
+                    "jdbc_name", name, "jdbc_prefix", prefix, "disable_security_manager", disableSecurityManager) });
+        } catch (final XmlRpcException exception) {
+            throw new IllegalStateException(
+                    "Failed add JDBC driver using ExaOperation. Cause: " + exception.getMessage(), exception);
+        }
     }
 
-    private void uploadJdbcDriver(final String jdbcDriverId, final File file) throws IOException, XmlRpcException {
+    private void uploadJdbcDriver(final String jdbcDriverId, final File file) throws IOException {
         try (final InputStream inputStream = new FileInputStream(file)) {
             final byte[] driverBytes = IOUtils.toByteArray(inputStream);
             this.client.execute(jdbcDriverId + ".uploadFile", new Object[] { driverBytes, file.getName() });
         } catch (final XmlRpcClientException exception) {
             // This exception happens always. I don't know why but it still works...
```

Comment:
Does exception still happens and this comment still valid?

Reply:
I think so ;-)